apps/wallet-mobile/src/components/Modal/ModalContext.tsx

```diff
@@ -6,10 +6,13 @@ type ModalState = {
   isOpen: boolean
   title: string
   content: React.ReactNode
+  isLoading: boolean
```

Comment:
I was tempted of doing that, if we were using redux to manage the requests I’d have gone for it, but the way it’s structured now by leveraging transitions states with react query it was not looking right, cuz it will reflect the query state only, basically 1:1 to query, also to block the screen the LoadingOverlay will have to sit on Modal container anyway. How are we preventing further interactions with the screen in other funnels that are leveraging modal ? (This will be fixed when we migrate to the modal wrapper component, but for that we will need to refactor the navigation)